test/test_lbex.m

```diff
@@ -0,0 +1,228 @@
+function test_patchsvd(datadirs)
```

Comment:
the function name here and in the other test script are swapped

Reply:
oeps